bin/utils.js

```diff
@@ -33,36 +31,51 @@ const readJson = (path) => {
 /**
  *
  */
-const jsonify = (data) =>
-	JSON.stringify(data, null, '\t');
+const jsonify = (data) => JSON.stringify(data, null, '\t');
 
 /**
  *
  */
 const writeJsonTo = (path, merge) => (data) => {
 	const oldData = readJson(path);
-	const newData = merge
-		? _.merge({}, oldData, data)
-		: data;
+	const newData = merge ? _.merge({}, oldData, data) : data;
 
 	fs.writeFileSync(path, jsonify(newData));
 };
 
 /**
  *
  */
-const logError = (error) =>
-	console.error(error);
-
+const logError = (error) => console.error(error);
 
+function withoutParagraph(html) {
+	return html.replace(/<p\b[^>]*>/gi, '').replace(/<\/p>/gi, '');
+}
 
 /**
- *
+ * @param {string} textElement
  */
+function objectToList(textElement) {
+	if (typeof textElement !== 'object') {
+		return textElement;
+	}
+	let list = ``;
+	for (const key in textElement) {
+		textElement.forEach((elem, index) => {
+			if (index === 0) {
+				list += `${elem}\n`;
+			}
+			list += ` * ${elem}\n`;
+		});
+	}
+	return list;
```

Comment:
Ici on n'utilise pas key, je pense que le foreach n'est pas utile.
On pourrait encore transformer ça en un `map` plus léger, par exemple :
```suggestion
return textElement.map((elem, index) =>
	index === 0 
		? `${elem}\n`
		: ` * ${elem}\n`
)
```
Si on déplace cette fonction dans le scraper, il faudrait aussi renommer le paramètre de `textElement` à quelque chose de plus explicite, par exemple `instructions` ?